### PR TITLE
Implemented CRUD, and crud testing

### PR DESCRIPTION
**Pull request by jvanboch**
1. Implemented the CUD in CRUD, and associated methods in the controller. 
2. Implemented JEST testing for the end points. There is an issue with the post and put tests not working, to be resolved later. 
3. Updated mysql and S3 login information.

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 const { Sequelize, DataTypes } = require('sequelize');
 
-const sequelize = new Sequelize( 'sbGallery', 'student', '', {
-  // host: 'localhost',
-  host: '54.153.5.233',
-  port: 3306,
+const sequelize = new Sequelize( 'sbGallery', 'student', 'Password!23', {
```
Comment (michellelockett) [inline database/index.js:3]: You should always be using environment variables that you do not check into github for all passwords, tokens, etc.

File: server/server.js [hunk 2]
```diff
@@ -13,10 +15,41 @@ app.get('/products/:shoeId/gallery', (req, res) => {
...
+      res.sendStatus(404);
+    });
+});
+
+//update
+app.put('/products/:shoeId/gallery/:imageId', jsonParser, (req,res)=>{
+  let {shoeId, imageId} = req.params;
+  controller.put.productImages(shoeId, imageId, req.body.imageUrl)
+    .then((data)=> {
+      res.sendStatus(404)
```
Comment (michellelockett) [inline server/server.js:38]: What status code do you intend to send in the case where the update was successful?

**Review by michellelockett — COMMENTED** [review]
Hi Jake!  Left a couple comments here.  Were you able to verify all of your routes work as expected when you hit them with Postman?

File: controller/index.js [hunk 3]
```diff
@@ -16,5 +16,32 @@ module.exports = {
...
+  },
+  post: {
+    productImages: (shoeId, imageUrl) => {
+      return new Promise((resolve, reject) => {
+        model.postShoeImages(shoeId, imageUrl)
+        .then(res=>console.log('posted'))
+        .catch(err=>console.log(err))
+      })
+    }
+  },
```
Comment (michellelockett) [inline controller/index.js:28]: I see you passing resolve and reject to the promise constructor, but I don't see you using them anywhere in your promises.  Let's chat about this in OH

File: server/server.js (hunk 2 above)
Comment (jvanboch) [reply server/server.js:38]: Hi  Michelle, 

I had updated to a res.sendStatus(204) for a successful. Didn't seem to update. Will be included in the pull request I make tonight.

File: controller/index.js (hunk 3 above)
Comment (jvanboch) [reply controller/index.js:28]: Hi Michelle, noticed my postman was getting hung up on my update, post, and delete request. I figured out it was due to this resolve issue. I have updated my controller file to resolve and reject for each method. Will be included in the pull request I do tonight.

File: database/index.js (hunk 1 above)
Comment (jvanboch) [reply database/index.js:3]: Updated and will be included in new pull request.

**Pull request merged**